lib/model/knex-migrator.js

```diff
@@ -23,7 +23,12 @@ const withKnex = (config) => (mutator) => {
 };
 
 // Given a database, initiates migrations on it.
-const migrate = (db) => db.migrate.latest({ directory: `${__dirname}/migrations` });
+// disableMigrationsListValidation: because knex-migrator now shares a DB table
+// with pg-migrator, knex's built-in check that all migrations found in the db
+// have corresponding migration files in `directory` will fail.
+// REVIEW note that we could maintain this check if we duplicate migration
+// tables and store pg migrations separately.
```

Comment:
This is only true if we don't move any existing migrations to the new migrator.

Reply:
For an existing install, will the new migrator fail if some migrations were executed by the old migrator?

Reply:
This should be tested with an e2e test or variation of the db-migration tests:

1. take db dump of migrated or semi-migrated legacy/knex-based db
2. import locally into postgres
3. fire up the migrator and see what happens

This might also provide a template for removing knex migrations.

Reply:
Done :heavy_check_mark: